packages/e2e/features/ui/components/authenticator/sign-in-with-username.feature

```diff
@@ -33,7 +33,9 @@ Feature: Sign In with Username
 
   # FORCE_CHANGE_PASSWORD tests are skipped as the temporary passwords used for these
   # test accounts will expire in Cognito.
+  @angular @react @vue @todo-angular @todo-vue
   Scenario: Sign in with force change password credentials
+    Given intercept '{ "headers": { "X-Amz-Target": "AWSCognitoIdentityProviderService.RespondToAuthChallenge" } }' with fixture "force-change-password"
```

Comment:
This is pretty easy to do! Use DevTools' Network tab, find a pattern to match on, and copy/paste 👍

Reply:
✨ ✨

Reply:
You know, we can probably mock even more things like `Auth.setupTOTP` responses, we can render QR code, and actually test if the image source contains the right source. That's just too exciting.